Neos.Eel/Tests/Unit/Helper/DateHelperTest.php

```diff
@@ -52,7 +53,7 @@ public function formatExamples()
         return [
             'DateTime object' => [$dateTime, 'Y-m-d H:i:s', '2013-07-03 12:34:56'],
             'timestamp as integer' => [1372856513, 'Y-m-d', '2013-07-03'],
-            'timestamp as string' => ['1372856513', 'Y-m-d', '2013-07-03'],
+            'timestamp as string' => ['@1372856513', 'Y-m-d', '2013-07-03'],
```

Comment:
@albe we discuss this with @kitsunet 
The problem what the same string '20160101' means completely different things if it date or timestamp. 
Timestamp as a string '20160101' is equivalent to: 22/08/1970 08:08:41 (format 'd/m/Y H:m:s')
Date as a string '20160101' is equivalent to: 01/01/2016 00:01:00 (format 'd/m/Y H:m:s')
IMO we must clearly separate these things.